test/goldens/end2end/lints-1.0.0.json_report.md

```diff
@@ -6,6 +6,22 @@
 
 * `pubspec.yaml` doesn't have a `repository` entry.
 
+<details>
+<summary>
+Failed to verify repository URL.
+</summary>
+
+Please provide a valid [`repository`](https://dart.dev/tools/pub/pubspec#repository) URL in `pubspec.yaml`, such that:
```

Comment:
Do we want to write this on a missing repository link? Closed source packages might not have any.

Reply:
I think if we are not giving points other packages get, we should give a reason and ways to fix it.
The source is available with the package, the license may restrict its use, but I don't think that should be a concern here.

Reply:
Are we giving less points when a repository is not present?

Reply:
As discussed offline, I've updated a code to only emit this if the verification explicitly failed. PTAL.